src/faction.cpp

```diff
@@ -217,6 +217,7 @@ std::string fac_ranking_text( int val )
 std::string fac_respect_text( int val )
 {
     // Respected, feared, etc.
+    //~ Faction respect - Legendary
     if( val >= 100 ) {
         return _( "Legendary" );
```

Comment:
For example, you'd want something like:
```suggestion
    if( val >= 100 ) {
        return pgettext( "Faction respect", "Legendary" );
```
(for consistency, I'd probably convert every faction respect value to use `pgettext`, not just the ones that were ambiguous, but I'm not a translator, so I'm not sure what is easiest for them)